rfd/00XX-agent-census.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: Vitor Enes (vitor@goteleport.com)
+state: draft
+---
+
+# RFD XX - Agent Census
+
+## Required Approvals
+
+* Engineering: @rjones && @jimbishopp
+* Product: @xin || @klizhentas
+
+## What
+
+This RFD details how we'll track more information about agents (aka Agent Census).
+A brief description of this task can be found in [Cloud's RFD 53](https://github.com/gravitational/cloud/tree/54559795b45b2e8515ea7e159d26cadfbb52482f/rfd/0053-prehog.md).
```

Comment:
Nit: maybe bring a few lines from RFD 53 to this RFD so we can get a little more context without switching over, & 
```suggestion
A brief description of this task can be found in [Cloud's RFD 53](https://github.com/gravitational/cloud/tree/54559795b45b2e8515ea7e159d26cadfbb52482f/rfd/0053-prehog.md#agent-census).
```